### PR TITLE
Add Artical Component to Storybook

### PR DESCRIPTION
**Pull request by colefriedlaender**
Artical Component

### CODE REVIEW

File: components/Artical/Artical.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Story, Meta } from "@storybook/react/types-6-0";
+import Artical, { ArticalProps } from "./Artical";
+
+export default {
+  title: "Common/Artical",
+  component: Artical,
```
Comment (fre-ben) [inline components/Artical/Artical.stories.tsx]: `Artical` is meant to be `Article`? :P

**Review by fre-ben — COMMENTED** [review]
LGTM 💯 
Maybe fix the `Article` typo if it's not meant on purpose to be `Artical`

**Review by phlgr — CHANGES_REQUESTED** [review]
You should fix the typo.

**Review by phlgr — COMMENTED** [review]
`Artical` seems to be a common misspelling of `Article`

**Pull request merged**